src/renderer/themes.css

```
@@ -417,6 +420,119 @@ it can be safely elided. This looks quite pleasant on this theme. */
+}
+
+.everforestDarkMedium {
+  --primary-text-color: #d3c6aa;
+  --secondary-text-color: #d3c6aa;
+  --tertiary-text-color: #d3c6aa;
+  --title-color: var(--accent-color);
+  --bg-color: #2d353b;
+  --favorite-icon-color: #0f0;
+  --card-bg-color: #232a2e;
+  --secondary-card-bg-color: #2d353b;
+  --scrollbar-color: #3f484e;
+  --scrollbar-color-hover: #46534f;
+  --side-nav-color: #232a2e;
+  --side-nav-hover-color: #131a1e;
+  --side-nav-active-color: #131a1e;
+  --search-bar-color: #343f44;
+  --logo-icon: url('../../_icons/iconEverforestLightMediumSmall.svg');
+  --logo-text: url('../../_icons/textEverforestLightMediumSmall.svg');
+}
+
+.everforestDarkLow {
+  --primary-text-color: #d3c6aa;
+  --secondary-text-color: #d3c6aa;
+  --tertiary-text-color: #d3c6aa;
+  --title-color: var(--accent-color);
+  --bg-color: #333c43;
+  --favorite-icon-color: #0f0;
+  --card-bg-color: #293136;
+  --secondary-card-bg-color: #333c43;
+  --scrollbar-color: #454f56;
+  --scrollbar-color-hover: #4d5b56;
+  --side-nav-color: #293136;
+  --side-nav-hover-color: #192126;
+  --side-nav-active-color: #192126;
+  --search-bar-color: #3a464c;
+  --logo-icon: url('../../_icons/iconEverforestLightLowSmall.svg');
+  --logo-text: url('../../_icons/textEverforestLightLowSmall.svg');
+}
+
+.everforestLightHard {
+  --primary-text-color: #3c4a52;
+  --secondary-text-color: #3c4a52;
+  --tertiary-text-color: #3c4a52;
+  --title-color: var(--accent-color);
+  --bg-color: #fffbef;
+  --favorite-icon-color: #0f0;
+  --card-bg-color: #f2efdf;
+  --secondary-card-bg-color: #fffbef;
+  --scrollbar-color: #d8d5c5;
+  --scrollbar-color-hover: #aeb5a2;
+  --side-nav-color: #f2efdf;
+  --side-nav-hover-color: #e2dfcf;
+  --side-nav-active-color: #e2dfcf;
+  --search-bar-color: #f8f5e4;
+  --logo-icon: url('../../_icons/iconEverforestDarkHardSmall.svg');
+  --logo-text: url('../../_icons/textEverforestDarkHardSmall.svg');
+}
+
+.everforestLightMedium {
+  --primary-text-color: #3c4a52;
+  --secondary-text-color: #3c4a52;
+  --tertiary-text-color: #3c4a52;
+  --title-color: var(--accent-color);
+  --bg-color: #fdf6e3;
+  --favorite-icon-color: #0f0;
+  --card-bg-color: #efe8d4;
+  --secondary-card-bg-color: #fdf6e3;
+  --scrollbar-color: #d0ccb7;
+  --scrollbar-color-hover: #adb39f;
+  --side-nav-color: #efe8d4;
+  --side-nav-hover-color: #dfd8c4;
+  --side-nav-active-color: #dfd8c4;
+  --search-bar-color: #f4f0d9;
+  --logo-icon: url('../../_icons/iconEverforestDarkMediumSmall.svg');
+  --logo-text: url('../../_icons/textEverforestDarkMediumSmall.svg');
+}
+
+.everforestLightLow {
+  --primary-text-color: #3c4a52;
+  --secondary-text-color: #3c4a52;
+  --tertiary-text-color: #3c4a52;
+  --title-color: var(--accent-color);
+  --bg-color: #f3ead3;
+  --favorite-icon-color: #0f0;
+  --card-bg-color: #eee8ce;
+  --secondary-card-bg-color: #f3ead3;
+  --scrollbar-color: #c8c3aa;
+  --scrollbar-color-hover: #a9b09b;
+  --side-nav-color: #e5dfc5;
```

Was really squinting when i saw this when testing but i think this also needs to be changed

```suggestion
  --side-nav-color: #eee8ce;
```